composer.json

```diff
@@ -10,11 +10,11 @@
         "race condition"
     ],
     "require": {
-        "php": "^5.6 || ^7.0"
+        "php": "^7.1"
```

Comment:
Would it make sense to drop PHP 7.1 support as well? It's EOL very soon.

Reply:
I wasn't sure about it, as many libraries are still on PHP 7.1, and I am not really using here any PHP 7.2 features. I've updated it only because PHP 7.1 is almost EOL and I am going to still support version 1.0.